quill/api/src/game.rs

```diff
@@ -109,7 +109,7 @@ impl Game {
     ///    println!("Found an entity with position {:?} and UUID {}", position, uuid);
     /// }
     /// ```
-    pub fn query<Q: Query>(&self) -> QueryIter<Q> {
+    pub fn query<Q: Query>(&mut self) -> QueryIter<Q> {
```

Comment:
Hmm I don't think quires should mutable borrow game, since you can write read only queries. It should be checked at runtime.

Reply:
I think for now it should be fine? *Theoretically* the new ECS will come eventually and fix this.